readme.md

```diff
@@ -115,24 +115,63 @@ The example assumes we merge `mps2021.1` into `mps2021.2`.
 5. Run all tests
 6. Update build model
 
-## Publishing and Releasing
 
-Run:
+## Publishing and releasing
```

Comment:
All the other headings are in all-caps variant, why change?

Reply:
Changed it back.